apps/awair/awair.star

```diff
@@ -0,0 +1,182 @@
+"""
+Applet: Awair
+Summary: Local Awair air data
+Description: Get air quality data from Awair's local API.
+Author: tabrindle
+"""
+
+load("encoding/json.star", "json")
+load("http.star", "http")
+load("render.star", "render")
+load("schema.star", "schema")
+
+def get_schema():
+    return schema.Schema(
+        version = "1",
+        fields = [
+            schema.Text(
+                id = "ip_address",
+                name = "IP Address of Awair device",
+                desc = "The local IP address of your Awair device.",
```

Comment:
Tidbyt apps run are hosted and run in the cloud, not on the devices themselves. Unless the Awair IP is on the public internet, the app won't be able to reach it.

Reply:
:-| I've been running this locally from my homebridge server for a few months, didn't even occur to me. I suppose this would still work if someone exposed their local API externally?

Reply:
Looks like it's oauth, so not trivial for a laymen to do. Exposing the local API externally works though.